api/config/config.example.json

```
@@ -45,5 +45,6 @@
   "secretKey": "super duper secret key",
   "GENERAL_API_URL_PROD": "http://mainendpoints:8080/api",
   "PERIPHERAL_API_URL_PROD": "http://sce-peripheral-api:8081/peripheralapi",
-  "MAILER_API_URL_PROD": "http://sce-cloud-api:8082/cloudapi"
+  "MAILER_API_URL_PROD": "http://sce-cloud-api:8082/cloudapi",
+  "DISCORD_PRINTING_KEY":"xxxxxxxxxxxxxxxx"
```

if we build out more features between our bot and core-v4 we will use this key again, maybe dont call it printing and instead

```suggestion
  "DISCORD_COREV4_KEY":"xxxxxxxxxxxxxxxx"
```


u can also skip and leave for when we actually do this stuff